### PR TITLE
Changed totalDayTimeInSeconds to Valheim's default value.

### PR DESCRIPTION
**Pull request by muro1214**
Valheim's total day and night time is 30 minutes.
Therefore, I changed totalDayTimeInSeconds from 1200 to 1800.

> Valheim days are 21 minutes long, with nights being 9 minutes long. A full loop is 30 minutes.

Source: https://valheim.fandom.com/wiki/Day_and_Night_Cycle

refs #154

### CODE REVIEW

**Review by Mixone-FinallyHere — COMMENTED** [review]
Not sure this is necessary but it can be done

**Review by radmint — DISMISSED** [review]

File: ValheimPlus/Initialize.cs [hunk 1]
```diff
@@ -9,11 +9,11 @@ namespace ValheimPlus
     // GITHUB REPOSITORY https://github.com/valheimPlus/ValheimPlus
     
 
-    [BepInPlugin("org.bepinex.plugins.valheim_plus", "Valheim Plus", "0.9.2")]
+    [BepInPlugin("org.bepinex.plugins.valheim_plus", "Valheim Plus", "0.9.2.1")]
     class ValheimPlusPlugin : BaseUnityPlugin
     {
         
-        public static string version = "0.9.2";
+        public static string version = "0.9.2.1";
```
Comment (radmint) [inline ValheimPlus/Initialize.cs:16]: Let's keep the version number unchanged from master

File: ValheimPlus/Initialize.cs [hunk 2]
```diff
@@ -36,17 +36,19 @@ void Awake()
...
-                if (!isUpToDate)
-                {
-                    Logger.LogError("There is a newer version available of ValheimPlus.");
-                    Logger.LogWarning("Please visit " + ValheimPlusPlugin.Repository + ".");
-                }
-                else
-                {
-                    Logger.LogInfo("ValheimPlus [" + version + "] is up to date.");
-                }
+                /*
```
Comment (radmint) [inline ValheimPlus/Initialize.cs:39]: Let's remove this commented change

**Review by radmint — CHANGES_REQUESTED** [review]
Thank you for your contribution! This PR is doing quite a few different things, would you mind separating out each feature into a different PR please?

**Review by xiaodoudou — CHANGES_REQUESTED** [review]
This should not be merged; as time is fixed on a different PR with a better approach.
See #172 172

**Comment by muro1214** [comment]
I'm sorry. This PR is closed and another one will be created.

**Comment by muro1214** [comment]
This changes is already merged different PR #172 
Thank you @xiaodoudou

**Pull request closed without merging**